Dangerfile

```diff
@@ -0,0 +1,82 @@
+
+has_app_changes = !git.added_files.grep(/Source.*\.swift/).empty?
+has_test_changes = !git.modified_files.grep(/Tests/).empty?
+
+###
+### Specific files to watch
+###
+
+# CI related files to watch for changes.
+@DANGER_CI_CD_FILES = ['.travis.yml', 'Dangerfile']
+
+# Dependency files to watch for changes.
+@DANGER_DEPENDENCY_FILES = ['Gemfile', 'JSQDataSourcesKit.podspec']
+
+# determine if any of the files were modified
+def didModify(files_array)
+	did_modify_files = false
+	files_array.each do |file_name|
+		if git.modified_files.include?(file_name) || git.deleted_files.include?(file_name)
+			did_modify_files = true
+		end
+	end
+	return did_modify_files
+end
+
+### WARNINGS
+
+warn('Changes to CI/CD files') if didModify(@DANGER_CI_CD_FILES)
+warn('Changes to dependency related files') if didModify(@DANGER_DEPENDENCY_FILES)
+warn("Big PR, try to keep changes smaller if you can") if git.lines_of_code > 500
+
+# Make it more obvious that a PR is a work in progress and shouldn't be merged yet
+warn("PR is classed as Work in Progress") if github.pr_title.include? "[WIP]"
+
+###
+### Auto label
+###
+
+# if github.pr_title.include? "[WIP]"
+# 	auto_label.wip=(github.pr_json["number"])
+# else
+# 	auto_label.remove("WIP")
+# 	# If you want to delete label
+# 	# auto_label.delete("WIP")
+# end
```

Comment:
let's remove L35 to here 😄

Reply:
NOTE: Was commenting out bc I needed the label on this repo + the danger token.  If you want to auto-label PR's for certain keywords, let me know ;)
Removed.